pages/en/lb3/Using-current-context.md

```diff
@@ -70,6 +70,33 @@ remoting metadata and set the `http` property to the special string value
 }
 ```
 
+For example:
+```js
+  MyObject.remoteMethod(
+    'myMethod',
+    {
+      http: {path: '/myMethod', verb: 'get'},
+      accepts: [
+        {arg: 'someArgument', type: 'string'},
+        {
+          arg: 'accessToken',
+          type: 'object',
+          http: function(ctx) {
+            return ctx.req.accessToken;
+          }
+        }
```

Comment:
Yes, if it's an alternative approach, it should be described separate from the existing documented approach.  Also, nnow that I read through this more closely, it might be worthwhile to add a bit more info to the existing section, e.g. clarify that the JSON block:
```
{
  "arg": "options",
  "type": "object",
  "http": "optionsFromRequest"
}
```
Goes in `common/models/my-model.json` (and perhaps show more of the surrounding JSON).

Also, can we explain why to use one approach vs. the other?